_inc/client/main.jsx

```diff
@@ -58,6 +58,9 @@ class Main extends React.Component {
 		window.addEventListener( 'beforeunload', this.onBeforeUnload );
 		// Handles transition between routes handled by react-router
 		this.props.router.listenBefore( this.routerWillLeave );
+
+		// Track initial page view
+		this.props.isSiteConnected && analytics.tracks.recordEvent( 'jetpack_wpa_page_view', { path: this.props.route.path } );
```

Comment:
personally - I find this code structures quite confusing. Using `if` statement makes it easier to parse